### PR TITLE
Create PR template

### PR DESCRIPTION
**Pull request by yuriy-fix**

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+- [ ] Bugfix
+- [ ] Feature
+
+## Checklist 
+
+- [ ] I have read the contribution guide: https://vaadin.com/docs-beta/latest/guide/contributing/overview/
+- [ ] I have added a description following the guideline.
+- [ ] The issue is created in the corresponding repository and I have referenced it.
+- [ ] I have added tests to ensure my change is effective and works as intended.
+- [ ] New and existing tests are passing locally with my change.
```
Comment (DiegoCardoso) [inline PULL_REQUEST_TEMPLATE.md:20]: Can/Should we mention the the new test should fail if the new code is removed?

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+- [ ] I have performed self-review and corrected misspellings.
+
+#### Additional for `Feature` type of change
+
+- [ ] Enhancement / new feature was discussed in a corresponding GitHub issue and Acceptance Criteria were created.
```
Comment (DiegoCardoso) [inline PULL_REQUEST_TEMPLATE.md:25]: Should it be on the singular?
```suggestion
- [ ] Enhancement / new feature was discussed in a corresponding GitHub issue and Acceptance Criteria was created.
```

**Review by DiegoCardoso — APPROVED** [review]

File: PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (yuriy-fix) [reply PULL_REQUEST_TEMPLATE.md:20]: It's specified within previous point: `I have added tests to ensure my change is effective and works as intended.`

Comment (DiegoCardoso) [reply PULL_REQUEST_TEMPLATE.md:20]: Oh, okay. Right! 👍

**Review by web-padawan — APPROVED** [review]
LGTM. I would add a link to acceptance criteria for enhancement, to explain what it is (and who should fill that).

**Comment by yuriy-fix** [comment]
Will be re-opened for monorepo.

**Pull request closed without merging**